lib/rspec/core/formatters/message_formatter.rb

```diff
@@ -0,0 +1,28 @@
+module RSpec
+  module Core
+    module Formatters
+      # @api private
+      # Formatter for providing message output as a fallback when no other
+      # profiler implements #message
+      class MessageFormatter
```

Comment:
Also, would it be worthwhile to subclass `BaseFormatter` so we get all the `output.sync` logic for this formatter as well?